packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -4,12 +4,20 @@
  * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
+/* istanbul ignore next */
```

Comment:
I'll look it up. `ignore next` seems to be used on an object-basis elsewhere in the package.